Resources/Catalog/FileCatalogProxyClient.py

```diff
@@ -31,6 +32,12 @@ def getName( self ):
     """
     return self.fcName
 
+  def setInterfaceMethods( self, methodTuple ):
+    self.interfaceMethods = methodTuple
+
+  def getInterfaceMethods( self ):
+    return self.interfaceMethods
+
```

Comment:
getters and setters are not pythonic. In this case interfaceMethods is a publc attribute of the object, you can access it directly. 
http://stackoverflow.com/questions/1554546/when-and-how-to-use-the-builtin-function-property-in-python